bitcoin-information/block-explorers.html

```diff
@@ -50,56 +50,44 @@ <h1>Bitcoin Block Explorers</h1>
   <section>
     <div class="container">
       <div class="row" style="font-weight:bold">
+        <!--LEFT COLUMN-->
         <div class="text-left col-xs-12 col-sm-6 col-lg-6 col-md-6">
           <h2 id="explorers"><a href="#explorers">Block Explorers:</a></h2>
           <ul>
-            <li><a href="https://platform.arkhamintelligence.com/" title="Arkham" target="_blank" rel="noopener">Arkham</a></li>
-            <li><a href="https://bitaps.com/" title="bitaps" target="_blank" rel="noopener">bitaps</a></li>
+            <li><a href="https://bitaps.com/" title="bitaps" target="_blank" rel="noopener">bitaps</a></li> <!--looks cool but has a link to a "crypto" telegram channel, it's their own channel-->
             <li><a href="https://bitcoinexplorer.org/" title="Explorer" target="_blank" rel="noopener">Bitcoin Explorer</a></li>
-            <li><a href="https://live.blockcypher.com/btc/" title="BlockCypher" target="_blank" rel="noopener">BlockCypher</a></li>
-            <li><a href="https://blockstream.info/" title="Blockstream" target="_blank" rel="noopener">Blockstream.info</a></li>
-            <li><a href="https://blocktime.site/" title="blocktime" target="_blank" rel="noopener">Blocktime</a> (search by date)</li>
-            <li><a href="https://mempool.space/blocks" title="mempool" target="_blank" rel="noopener">Mempool.space</a></li>
-            <li><a href="https://openoms.github.io/blyss-block-explorer-client/" title="Spiral" target="_blank" rel="noopener">Spiral</a> (strong privacy)</li>
-            <li><a href="https://sochain.com/" title="SoChain" target="_blank" rel="noopener">SoChain</a></li>
+            <li><a href="https://blocktime.site/" title="blocktime" target="_blank" rel="noopener">Blocktime</a> (search by date)</li> <!--links to mempool.space for block info-->
+            <li><a href="https://mempool.space" title="mempool.space" target="_blank" rel="noopener">Mempool.space</a></li> <!--keeping it for now because they removed liquid-->
+            <li><a href="https://mempool.guide" title="WE LOVE BITCOIN" target="_blank" rel="noopener">Mempool.guide</a></li> 
+            <li><a href="https://openoms.github.io/blyss-block-explorer-client/" title="Spiral" target="_blank" rel="noopener">Spiral</a> (strong privacy)</li> <!--not verified, cool if true but missing disclaimers-->
             <li><a href="https://yogh.io/" title="Yogh" target="_blank" rel="noopener">Yogh.io</a> - raw data block explorer</li>
           </ul>
           <h2 id="chainalysis"><a href="#chainalysis">Blockchain Analysis:</a></h2>
           <ul>
-            <li><a href="https://www.anchain.ai/" title="AnChain.ai" target="_blank" rel="noopener">AnChain.ai</a></li>
```

Comment:
Do we really need to remove every chain analysis link because they also do chain analysis on shitcoins? I don't see how that harms our users.